exchanges/order/order_types.go

```diff
@@ -350,9 +350,9 @@ const (
 // Type enforces a standard for order types across the code base
 type Type uint32
 
-// Is checks to see if the enum contains the flag
-func (t Type) Is(in Type) bool {
-	return in != 0 && t&in == in
+// Is checks to see if the Type contains the Type cmp
+func (t Type) Is(cmp Type) bool {
+	return cmp != 0 && t&cmp == cmp
 }
```

Comment:
🚧 With the change to not using "Type.Is()` in `Type.String()`, I don't think this is used anymore.
```suggestion
```

Reply:
I added a unit test for this and I believe it is better to keep it than to remove as it has relevance.

Reply:
I think it only has relevance if we have hybrid order types, which leads me to my only outstanding issue now.